src/main/java/uk/gov/hmcts/reform/wataskmonitor/config/SnakeCaseFeignConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.wataskmonitor.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.codec.Decoder;
+import feign.codec.Encoder;
+import feign.form.spring.SpringFormEncoder;
+import org.springframework.beans.factory.ObjectFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.http.HttpMessageConverters;
+import org.springframework.cloud.openfeign.support.ResponseEntityDecoder;
+import org.springframework.cloud.openfeign.support.SpringDecoder;
+import org.springframework.cloud.openfeign.support.SpringEncoder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+
+@Configuration
+@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+public class SnakeCaseFeignConfiguration {
```

Comment:
Is this now defaulting everything to snake_case?  
Just note that the Task Monitor will interact with Camunda which will return `camelCase` and we should be mapping that.

Reply:
This will make any feign interfaces snake case that we define with     
configuration = SnakeCaseFeignConfiguration.class